test/state/host.cpp

```diff
@@ -266,6 +266,17 @@ evmc::Result Host::execute_message(const evmc_message& msg) noexcept
 
     // Copy of the code. Revert will invalidate the account.
     const auto code = dst_acc != nullptr ? dst_acc->code : bytes{};
+
+    if (msg.kind == EVMC_DELEGATECALL)
```

Comment:
This has to be changed since it was decided on some EOF calls that we want to only block EOF->legacy delegatecall.

Reply:
This commit should be removed.